### ISSUE TITLE
NPE when opening any application folder from folders panel

### ISSUE THREAD

**Issue by oshiryaeva** [cuba-platform/cuba#1830]
### Environment

- Platform version: 7.0.0.BETA4

### Description of the bug

- Minimal reproducible example
1. Setup app properties to display folders panel:
```
cuba.web.foldersPaneEnabled = true
cuba.web.foldersPaneVisibleByDefault = true
```
2. Open any application screen
3. Save it as application folder
4. Open it from folders panel

- Expected behavior
Screen is opened
- Actual behavior
The screen is opened, but also NPE occurs:

```java
java.lang.NullPointerException
	at com.haulmont.cuba.gui.components.AbstractWindow.getComponent(AbstractWindow.java:267)
	at com.haulmont.cuba.gui.components.ComponentContainer.getComponentNN(ComponentContainer.java:92)
	at com.haulmont.cuba.web.app.folders.FoldersBean.openFolder(FoldersBean.java:95)
	at com.haulmont.cuba.web.app.folders.CubaFoldersPane.openFolder(CubaFoldersPane.java:626)
	at com.haulmont.cuba.web.app.folders.CubaFoldersPane$FolderClickListener.itemClick(CubaFoldersPane.java:709)
```

**Comment by jreznot** [comment]
@natfirst It seems like it is already fixed in 7.0.0

**Comment by oshiryaeva** [comment]
Re-tested in 7.0.0, it is fixed.
